run_cibuildwheel_on_ec2.sh

```diff
@@ -0,0 +1,27 @@
+if [ $# -ne 6 ]; then
+    echo "Invalid number of parameters, you need to provide role name, region name, bucket name, prefix, express region name and express bucket name"
+    echo "Usage: $0 S3RoleName us-west-2 s3torchconnector-test-bucket-name prefix-name/ us-east-1 s3torchconnectorclient-express-bucket-name"
+    exit 1
+fi
+
+ROLE_NAME=$1
+REGION_NAME=$2
+BUCKET_NAME=$3
+PREFIX=$4
+EXPRESS_REGION_NAME=$5
+EXPRESS_BUCKET_NAME=$6
+
+FILE_NAME="tmp_cred.json"
+TOKEN=`curl -s -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600"` && curl -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/meta-data/iam/security-credentials/${ROLE_NAME} >> ${FILE_NAME}
```

Comment:
Can we please think of a nicer way to write this?

Reply:
> Can we please think of a nicer way to write this?

Could you please explain the concept of `nicer` in that case? Is it to remove using of curl or splitting that command in few smaller steps?

Reply:
I am not particularly fond of using the IP address

Reply:
> I am not particularly fond of using the IP address

That ip address is official way to access the EC2 instance metadata from within instance itself: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html